src/components/organisms/AssetContent/EditHistory.module.css

```diff
@@ -7,6 +7,7 @@
 .history {
   font-size: var(--font-size-small);
   padding-left: 1.25rem;
+  margin-bottom: calc(var(--spacer) );
```

Comment:
it looks better without this bottom spacing I think. Then bottom & left white space of content box is more consistent

Reply:
Ok then, I'll remove the bottom spacing